src/kili/llm/services/export/dynamic.py

```diff
@@ -173,6 +179,9 @@ def _build_rounds(self, chat_items, annotations, json_interface):
                         has_children = True
                         parent_target = chat_item["id"]
 
+                current_round["annotations"] += [
```

Comment:
we'll need to do this if we have a pending round when the loop finish (l203, before the ` rounds.append(current_round)`)

Reply:
Change it thanks